docs/uploads.md

```diff
@@ -1,7 +1,7 @@
 # File Uploads
-File uploads are supported via Amazon S3. 
+File uploads are supported via Amazon S3 or direct uploading. 
```

Comment:
The temporary file thing isn't even being utilized anymore so that might even be able to be removed, but I'll try and clean up the documentation a bit and add a bit more about local uploading to this file